### PR TITLE
add documentation inline to env vars files

### PR DESCRIPTION
**Pull request by kmoscoe**
Instead of making users go back and forth from doc to env vars files, put the doc inline to make editing more convenient. 
I've tested that this works with local and remote DBs.

### CODE REVIEW

File: custom_dc/cloudsql_env.list [hunk 1]
```diff
@@ -1,12 +1,24 @@
+# (Optional) Secret token to authorize users to perform /admin page operations.
+ADMIN_SECRET=
+# The name of the Cloud SQL instance, in the form <PROJECT_ID>:<LOCATION>:<INSTANCE_ID>.     
+CLOUDSQL_INSTANCE=
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
```
Comment (keyurva) [inline custom_dc/cloudsql_env.list:6]: Can we do logical grouping of the vars? e.g.

* Keys: DC_API_KEY, MAPS_API_KEY
* DB: USE_CLOUD_SQL, CLOUDSQL_INSTANCE, DB_NAME, DB_USER, DB_PASS
* NL: ENABLE_MODEL
* Others: FLASK_ENV, ADMIN_SECRET, REDIS_HOST

Similarly for the other file.

**Comment by kmoscoe** [comment]
I actually did it in alphabetical order. But if you think by category is
better, I can definitely do it.

**Comment by keyurva** [comment]
Yes, can we do by category?

**Comment by kmoscoe** [comment]
Yes, I did.

File: custom_dc/cloudsql_env.list [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+
+# API key for accessing the base Data Commons API.
 DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
 MAPS_API_KEY=
-GCS_DATA_PATH=
+# Use Google Cloud SQL as the database. 
+USE_CLOUDSQL=true
+# The name of the Cloud SQL instance, in the form <PROJECT_ID>:<LOCATION>:<INSTANCE_ID>.     
 CLOUDSQL_INSTANCE=
```
Comment (keyurva) [inline custom_dc/cloudsql_env.list:9]: The `DB_*` vars belong to this logical group. Consider moving them under here.

**Review by keyurva — APPROVED** [review]
Thanks!

**Pull request merged**